openfl/transport/grpc/director_server.py

```diff
@@ -72,7 +72,8 @@ def get_caller(self, context):
 
     def start(self):
         """Launch the director GRPC server."""
-        loop = asyncio.get_event_loop()   # TODO: refactor after end of support for python3.6
+        loop = asyncio.get_event_loop()  # TODO: refactor after end of support for python3.6
+        loop.create_task(self.director.run_background_tasks())
```

Comment:
It really should be a task?

Reply:
Please give me your proposal

Reply:
```
self.director.run_background_tasks()
```
`run_background_tasks` can be static, it only creates tasks in event loop for each background_task.

Reply:
was moved to director_server